engine/src/main/java/org/hibernate/search/engine/metadata/impl/SortableFieldMetadata.java

```diff
@@ -21,6 +21,7 @@ private SortableFieldMetadata(String fieldName) {
 
 	/**
 	 * The name of the field whose field bridge will be used to determine the value for the doc value field.
+	 * @return the name of the field
```

Comment:
I'm pretty sure it's an error in this case.
As far as I understand, one cannot add a description withouth also describing the  parameters in the signature.

I will check again, just in case.